src/test/jest/extension.test.ts

```diff
@@ -8,7 +8,11 @@ jest.mock('../../NoteWorkspace');
 
 beforeEach(() => {
   NoteWorkspace.cfg = () => {
-    return NoteWorkspace.DEFAULT_CONFIG;
+    let config =  NoteWorkspace.DEFAULT_CONFIG;
```

Comment:
If we are keeping the `pipedWikiLinksSyntax` can you add some test cases for the non-default config?

Also may be worth having a test for use of something else as `pipedWikiLinksSeparator` as well as an invalid `pipedWikiLinksSeparator`

Reply:
> If we are keeping the pipedWikiLinksSyntax can you add some test cases for the non-default config?
Sure - I sadly don't have time to do this tonight, I'll get to it tomorrow.

> Also may be worth having a test for use of something else as `pipedWikiLinksSeparator`
Will do!

> as well as an invalid `pipedWikiLinksSeparator`

This is something we can prevent from happening by using a good pattern within `package.json`:

```json
{
    "pattern": "[^\\[\\]]+",
    "patternErrorMessage": "Warning: This separator will unfortunately not work - It would break the wikilink syntax."
}
```
This blocks a user from ever setting "[" or "]" as a separator.